src/Test.Utilities/DiagnosticAnalyzerTestBase.cs

```diff
@@ -216,78 +216,98 @@ private static DiagnosticResultLocation[] ParseResultLocations(string defaultPat
 
         protected void VerifyCSharpUnsafeCode(string source, params DiagnosticResult[] expected)
         {
-            Verify(new[] { source }.ToFileAndSource(), LanguageNames.CSharp, GetCSharpDiagnosticAnalyzer(), DefaultTestValidationMode, true, expected);
+            Verify(new[] { source }.ToFileAndSource(), LanguageNames.CSharp, GetCSharpDiagnosticAnalyzer(), DefaultTestValidationMode, true, null, expected);
```

Comment:
>null [](start = 141, length = 4)

there same as above and below: parameter name will be helpful for null/true/false